avm/res/network/load-balancer/backend-address-pool/README.md

```diff
@@ -0,0 +1,101 @@
+# Load Balancer Backend Address Pools `[Microsoft.Network/loadBalancers/backendAddressPools]`
+
+This module deploys a Load Balancer Backend Address Pools.
+
+## Navigation
+
+- [Resource Types](#Resource-Types)
+- [Parameters](#Parameters)
+- [Outputs](#Outputs)
+- [Cross-referenced modules](#Cross-referenced-modules)
+
+## Resource Types
+
+| Resource Type | API Version |
+| :-- | :-- |
+| `Microsoft.Network/loadBalancers/backendAddressPools` | [2023-04-01](https://learn.microsoft.com/en-us/azure/templates/Microsoft.Network/2023-04-01/loadBalancers/backendAddressPools) |
+
+## Parameters
+
+**Required parameters**
+
+| Parameter | Type | Description |
+| :-- | :-- | :-- |
+| [`name`](#parameter-name) | string | The name of the backend address pool. |
+
+**Conditional parameters**
+
+| Parameter | Type | Description |
+| :-- | :-- | :-- |
+| [`loadBalancerName`](#parameter-loadbalancername) | string | The name of the parent load balancer. Required if the template is used in a standalone deployment. |
+
+**Optional parameters**
+
+| Parameter | Type | Description |
+| :-- | :-- | :-- |
+| [`drainPeriodInSeconds`](#parameter-drainperiodinseconds) | int | Amount of seconds Load Balancer waits for before sending RESET to client and backend address. if value is 0 then this property will be set to null. Subscription must register the feature Microsoft.Network/SLBAllowConnectionDraining before using this property. |
+| [`enableTelemetry`](#parameter-enabletelemetry) | bool | Enable telemetry via a Globally Unique Identifier (GUID). |
+| [`loadBalancerBackendAddresses`](#parameter-loadbalancerbackendaddresses) | array | An array of backend addresses. |
+| [`syncMode`](#parameter-syncmode) | string | Backend address synchronous mode for the backend pool. |
+| [`tunnelInterfaces`](#parameter-tunnelinterfaces) | array | An array of gateway load balancer tunnel interfaces. |
+
+### Parameter: `drainPeriodInSeconds`
+
+Amount of seconds Load Balancer waits for before sending RESET to client and backend address. if value is 0 then this property will be set to null. Subscription must register the feature Microsoft.Network/SLBAllowConnectionDraining before using this property.
+- Required: No
+- Type: int
+- Default: `0`
+
+### Parameter: `enableTelemetry`
+
+Enable telemetry via a Globally Unique Identifier (GUID).
+- Required: No
+- Type: bool
+- Default: `True`
+
+### Parameter: `loadBalancerBackendAddresses`
+
+An array of backend addresses.
+- Required: No
+- Type: array
+- Default: `[]`
+
+### Parameter: `loadBalancerName`
+
+The name of the parent load balancer. Required if the template is used in a standalone deployment.
+- Required: Yes
+- Type: string
+
+### Parameter: `name`
+
+The name of the backend address pool.
+- Required: Yes
+- Type: string
+
+### Parameter: `syncMode`
+
+Backend address synchronous mode for the backend pool.
+- Required: No
+- Type: string
+- Default: `''`
+- Allowed: `['', Automatic, Manual]`
```

Comment:
It apperas that the used readme utility was not the latest as this line shoud now be split across multiple lines